sharding/sharding.go

```diff
@@ -240,7 +252,31 @@ func (r *ShardingFerry) deltaCopyJoinedTables() error {
 		}
 	}
 
-	return r.Ferry.RunStandaloneDataCopy(tables)
+	err := r.Ferry.RunStandaloneDataCopy(tables)
+	if err != nil {
+		return err
+	}
+
+	verifier, err := r.newIterativeVerifier()
+	if err != nil {
+		return err
+	}
+
+	verifier.Tables = tables
+
+	err = verifier.Initialize()
+	if err != nil {
+		return err
+	}
+
+	verificationResult, err := verifier.VerifyOnce()
+	if err != nil {
+		return err
+	} else if !verificationResult.DataCorrect {
```

Comment:
nitpick: Can you make this just an `if` instead of an `else if` (because of the `return`, that's equivalent here, but I find it a tiny bit easier to understand).